jbpm-human-task/jbpm-human-task-audit/src/main/java/org/jbpm/services/task/audit/JPATaskLifeCycleEventListener.java

```diff
@@ -700,7 +701,12 @@ public void beforeTaskNotificationEvent(TaskEvent event) {
     public void afterTaskNotificationEvent(TaskEvent event) {
 
     }
-
+    
+    @Override
+    public void beforeTaskOutputVariableChangedEvent(TaskEvent event, Map<String, Object> variables){
+        
+    }
+    
```

Comment:
Hmm, just thinking why is this overridden since we have a `default` construct in the interface.